### ISSUE TITLE
As a Project Coordinator, I want to be able to be able to track project location information so that I know exactly where a given project is located. 

### ISSUE THREAD

**Issue by LindsayMacfarlane** [bcgov/cas-cif#578]
#### Description:

As a Project Coordinator, I want to be able to be able to track project location information so that I know exactly where a given project is located. 

#### Acceptance Criteria:

- [ ] Project Facility. input = text
- [ ] BCGHGID. input = number. 
     - Format: _ - _ _ _ _ _ _ - _ _ _ _ or 11 digits
     - Breakdown of the the format: Sector - NAICS # - Cumulative Digit
     - Sector is an internally created reference: 1 = Linear Facility Operation, 2 = Single Facility Operation, 3 = Electricity import
     - Could be imported from SWRS
- [ ] Longitude
- [ ] Latitude

#### Development Checklist:

- [ ] Checklist item
- [ ] Checklist item
- [ ] Checklist item
- [ ] Meets the DOD

#### Notes:

- E.g., location is gps coordinates, could also include street address. 

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

**Comment by LindsayMacfarlane** [comment]
@elliottSlee - did project location information come up in any of our discussions or usability testing? I feel location (e.g., street address, gps coordinates) are tracked in CIF spreadsheets and I remember it being mentioned during initial discovery and yet, I don't remember this being flagged as something missing during our usability testing sessions.

**Comment by LindsayMacfarlane** [comment]
FYI @nanyangpro - we might need a conversation with the tech team related to project information. I believe this relates to our conversation with Taylor on the 'other data' that CIF would like to see included in the system.

**Comment by nanyangpro** [comment]
Good call, @LindsayMacfarlane. Yes, we do needs the tech team's perspectives on "other data" important to CIF.

- When would the tech team become available again?
- This geo information is an administrative piece to me. Maybe it's an Ops Team conversation instead?

**Comment by LindsayMacfarlane** [comment]
Based on CIF prioritization session on Aug 16, 2022, CIF team agrees this is a low priority and can remain in the icebox for now.
